tests/findings-to-issues.test.sh

```
@@ -0,0 +1,183 @@
+echo "### Suite 5: defer_issues_enabled=false short-circuits identically"
+
+assert_in_section "$REVIEW_PR_MD" 'Phase 2.5|findings-to-issues' 'Phase 3|Final Aggregation' \
+  'uberdev_read_enum.*defer_issues_enabled|defer_issues_enabled.*uberdev_read_enum' \
+  'C1 review-pr.md reads defer_issues_enabled via uberdev_read_enum'
+assert_in_section "$REVIEW_PR_MD" 'Phase 2.5|findings-to-issues' 'Phase 3|Final Aggregation' \
+  'effective.*enabled|both.*ON|CLI.*AND.*config' \
+  'C2 effective-enabled gate is AND of CLI flag and config key'
+
+# Runtime smoke: source lib/config-read.sh in fixture context with key absent, then with key=false.
+if [[ -f "$REPO_ROOT/plugins/uberdev/lib/config-read.sh" ]]; then
+  pushd "$REPO_ROOT" >/dev/null
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)